### PR TITLE
Fix YugenMangas

### PR DESCRIPTION
**Pull request by wolfghost9898**
Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Review by alessandrojean — CHANGES_REQUESTED** [review]

File: .run/MadaraGenerator.run.xml [hunk 1]
```diff
@@ -1,13 +1,8 @@
 <component name="ProjectRunConfigurationManager">
```
[the remark this answers is not in the document]
Comment (wolfghost9898) [reply .run/MadaraGenerator.run.xml]: sorry fixed

File: .run/MadaraGenerator.run.xml [hunk 2]
```diff
@@ -14,4 +14,4 @@
       <option name="Gradle.BeforeRunTask" enabled="true" tasks="ktLint" externalProjectPath="$PROJECT_DIR$/multisrc" vmOptions="" scriptParameters="" />
     </method>
   </configuration>
-</component>
\ No newline at end of file
+</component>g
```
Comment (alessandrojean) [inline .run/MadaraGenerator.run.xml:17]:
```suggestion
</component>
```

**Review by alessandrojean — CHANGES_REQUESTED** [review]

File: multisrc/overrides/madara/yugenmangas/src/YugenMangas.kt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package eu.kanade.tachiyomi.extension.es.yugenmangas
+
+import eu.kanade.tachiyomi.multisrc.madara.Madara
+
+class YugenMangas : Madara("YugenMangas", "https://yugenmangas.com/", "es")
```
Comment (arkon) [inline multisrc/overrides/madara/yugenmangas/src/YugenMangas.kt:5]: I don't think this file is actually needed if it's not overriding anything special.

Comment (Clouddark75) [reply multisrc/overrides/madara/yugenmangas/src/YugenMangas.kt:5]: Just adding the source to Madara won't do, the site is using a heavily modified version of it, like Reaper kinda mod. That will need more work to have a working extension for that site.

**Comment by CarlosEsco** [comment]
So is this good?

**Comment by arkon** [comment]
@wolfghost9898 Was this tested?

**Comment by wolfghost9898** [comment]
> @wolfghost9898 Was this tested?
yes, I've been using it since I did the PR

image

**Pull request merged**